sopel/modules/help.py

```diff
@@ -12,10 +12,81 @@
 
 import collections
 import requests
+import socket
 import textwrap
 
 from sopel.logger import get_logger
 from sopel.module import commands, rule, example, priority
+from sopel.config.types import (
+    StaticSection, ChoiceAttribute
+)
+
+
+# Pastebin handlers
+
+
+def post_to_clbin(bot, msg):
+    result = requests.post('https://clbin.com/', data={'clbin': msg})
+    result = result.text
+
+    if 'https://clbin.com/' in result:
+        return result
+    else:
+        LOGGER.error("Invalid result %s", result)
+        raise PostingException()
+
+
+def post_to_0x0(bot, msg):
+    payload = {'file': msg}
+    result = requests.post('https://0x0.st', files=payload)
+    return result.text
+
+
+def post_to_hastebin(bot, msg):
+    result = requests.post('https://hastebin.com/documents', data=msg)
+    result = result.json()
```

Comment:
If an invalid response is returned, this line will throw a `ValueError: NO JSON...`, which is not handled nicely currently. See other comments, as it may just be better to do a larger change re: error handling.